x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlQueryRequest.java

```diff
@@ -180,6 +180,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (clientId() != null) {
             builder.field("client_id", clientId());
         }
+        if (version() != null) {
+            builder.field("client_version", version().toString());
```

Comment:
Some constants for these strings would be nice.

Reply:
Very much agreed. I would address this in a follow-up PR, thought, since these hardcoded strings are used extensively in tests?